storage/src/main/java/com/tencent/rss/storage/util/StorageType.java

```diff
@@ -22,5 +22,6 @@
   HDFS,
   LOCALFILE,
   FILE,
-  LOCALFILE_AND_HDFS
+  LOCALFILE_AND_HDFS,
+  MEMORY
```

Comment:
MEMORY shouldn't be a single storage

Reply:
will update storageType with possible combination